app/views/digitalpack/info.scala.html

```diff
@@ -1,88 +1,110 @@
 @(edition: DigitalEdition)
 
-@main(if(edition.id == "uk") {
-    "Digital | Subscriptions | The Guardian"
-} else {
-    s"Digital - ${edition.name} | Subscriptions | The Guardian"
-}) {
+@main(
+    title = if(edition.id == "uk") "Digital | Subscriptions | The Guardian" else s"Digital - ${edition.name} | Subscriptions | The Guardian",
+    bodyClassName = "digital-pack") {
+<div class="gs-container dp-container">
```

Comment:
What's the thinking of mixing `dp-container` with `gs-container`, feels like there's a lot of overlap / conflict. Is there a simpler way of writing this?